src/sonic-yang-mgmt/tests/libyang-python-tests/test_sonic_yang.py

```diff
@@ -266,20 +266,38 @@ def test_get_leafref_type_schema(self, yang_s, data):
             data_type = yang_s.get_leafref_type_schema(xpath)
             assert expected_type == data_type
 
-    def test_xlate_rev_xlate(self):
+    """
+    This is helper function to load YANG models for tests cases, which works
+    on Real SONiC Yang models. Mainly tests  for translation and reverse
+    translation.
+    """
+    @pytest.fixture(autouse=True, scope='class')
+    def sonic_yang_data(self):
+        sonic_yang_dir = "/sonic/src/sonic-yang-models/yang-models/"
+        sonic_yang_test_file = "/sonic/src/sonic-yang-models/tests/yang_model_tests/yangTest.json"
+
+        syc = sy.sonic_yang(sonic_yang_dir)
+        syc.loadYangModel()
+
+        sonic_yang_data = dict()
+        sonic_yang_data['yang_dir'] = sonic_yang_dir
+        sonic_yang_data['test_file'] = sonic_yang_test_file
+        sonic_yang_data['syc'] = syc
+
+        return sonic_yang_data
+
+    def test_xlate_rev_xlate(self, sonic_yang_data):
         # In this test, xlation and revXlation is tested with latest Sonic
         # YANG model.
+        test_file = sonic_yang_data['test_file']
+        syc = sonic_yang_data['syc']
 
-        yang_dir = "/sonic/src/sonic-yang-models/yang-models/"
-        yang_test_file = "/sonic/src/sonic-yang-models/tests/yang_model_tests/yangTest.json"
-        jIn = self.readIjsonInput(yang_test_file, 'SAMPLE_CONFIG_DB_JSON')
-        # load yang models
-        syc = sy.sonic_yang(yang_dir)
-
-        syc.loadYangModel()
+        jIn = self.readIjsonInput(test_file, 'SAMPLE_CONFIG_DB_JSON')
 
         syc.load_data(json.loads(jIn))
 
+        # TODO: Make sure no extra table is loaded
```

Comment:
Can you elaborate the missing function here?

Reply:
Right now, In test file (i.e. sample config) we have extra tables, for which yang models are not merged yet: 
https://github.com/Azure/sonic-buildimage/pull/4442.

This line is to remind, once all yang models are merged, I can even test here that no extra tables exist in test config.